src/content/docs/apm/agents/php-agent/configuration/distributed-tracing-php-agent.mdx

```diff
@@ -0,0 +1,616 @@
+
+### Step 1. Identify services
+
+Figure out which services touch your request so you can instrument each of them to send trace data to New Relic.
+
+### Step 2. Instrument each service with an APM agent [#update-agents]
+
+You'll repeat the agent installation routine for each service involved in your transactions. If some of your services use other languages, simply repeat the [installation steps](/docs/distributed-tracing/enable-configure/quick-start) for those languages.
+
+To start the installation routine, click the tile below. When you are finished installing each agent, return here to see tips for [viewing your traces](#view-traces).
+
+<TechTileGrid>
+  <TechTile
+    name="APM: PHP"
+    variant="TechTile"
+    to="https://one.newrelic.com/launcher/nr1-core.settings?pane=eyJuZXJkbGV0SWQiOiJ0dWNzb24ucGxnLWluc3RydW1lbnQtZXZlcnl0aGluZyJ9&cards[0]=eyJuZXJkbGV0SWQiOiJzZXR1cC1uZXJkbGV0cy5zZXR1cC1waHAtaW50ZWdyYXRpb24iLCJhY2NvdW50SWQiOjI2NDA0MDl9&platform[accountId]=2498654&platform[timeRange][duration]=1800000&platform[$isFallbackTimeRange]=true"
+    icon={<img src={phplogo} alt="PHP"/>}
+  />
+</TechTileGrid>
+
+### Step 3. View traces [#view-traces]
+
+After you instrument each of your services with APM agents, generate some traffic in your application so we can capture some traces. Here are some ways to view your traces in the UI:
+
+<CollapserGroup>
+  <Collapser
+    id="entity-explorer"
+    title="View traces that include a specific service"
+  >
+    Here's one way you can see traces for a particular service:
+
+    1. Go to **[one.newrelic.com](https://one.newrelic.com/)**.
+    2. Click **APM** in the top menu bar.
+    3. Click your service.
+    4. In the left navigation's **Monitor** section, click **Distributed tracing**.
+    5. If you don't see the traces you want, you can filter by the `trace.id`.
+  </Collapser>
+
+  <Collapser
+    id="dt-launcher"
+    title="View traces across accounts"
+  >
+    This option allows you to search all traces across all New Relic accounts in your organization that you have access to.
+
+    1. Go to **[one.newrelic.com](https://one.newrelic.com/)**.
+    2. Click **Browse data** in the top menu bar, and then click **Traces**.
+    3. Select your entity in the left pane.
+    4. If you don't see the traces you want, you can filter by the `trace.id`.
+  </Collapser>
+
+  <Collapser
+    id="logs-context"
+    title="Examine logs for trace details"
+  >
+    You can bring your logs and application's data together to make troubleshooting easier and faster. With [logs in context](/docs/logs/logs-context/configure-logs-context-apm-agents/), you can see log messages related to your errors and traces directly in your app's UI.
```

Comment:
We are currently working towards logs in context on the PHP agent, but it isn't released yet. Is this a section that we could add later? Otherwise the PHP page looks good, thanks!

Reply:
Logs in context is now available for the PHP agent.